openstates/de/legislators.py

```diff
@@ -16,27 +16,50 @@ def scrape(self, chamber, term, text=methodcaller('text_content'),
                re_spaces=re.compile(r'\s{,5}')):
```

Comment:
Remove the non-standard parameters to this function (`text` and `re_spaces`). This should also allow you to remove some unused dependencies. @paultag suggested that these parameters are probably from someone trying to run the scraper outside of `billy`, which is a really odd use-case.